routes/api/controllers/manageUsers.js

```diff
@@ -23,6 +26,8 @@ const signupUser = async (req, res, next) => {
         const result = await registerUser(body);
         if (result && result !== 409 && result.status !== 400) {
             await result.save();
+            sendingEmails(result.email, result.verificationToken);
```

Comment:
Czytam i zastanawiam się czy specjalnie pominęłaś await czy nie? :) dla osoby czytającej może to być mylące, gdyż ja oczekuję że to będzie funkcja synchroniczna a tymczasem ona jest asynchroniczna. W takich sytuacjach dobrze jest dodać komentarz na górze zaraz przed tą linijką który mówi że Tu odbywa się działanie asynchroniczne na które nie czekamy.